### PR TITLE
feat: add skeleton-placeholder class

### PR DESCRIPTION
**Pull request by ryanomackey**
Adds a new `.bx--skeleton__placeholder` class for use with a new React component.

#### Changelog

**New**

- Adds a new class for the <SkeletonPlaceholder /> component.

**Changed**

- N/A

**Removed**

- N/A

#### Testing / Reviewing

Can be tested along with the React PR: https://github.com/IBM/carbon-components-react/pull/1401

### CODE REVIEW

**Comment by tw15egan** [comment]
@ryanomackey Can you fix this PR so that it passes Travis? Seems like `skeleton` uses the `$skeleton` variable, so this new file will need to import the `theme.scss` file 

`@import '../../globals/scss/theme';`

File: src/components/skeleton/_skeleton-placeholder.scss [hunk 1]
```diff
@@ -0,0 +1,9 @@
+@import '../../globals/scss/helper-mixins';
+@import '../../globals/scss/import-once';
+@import '../../globals/scss/vars';
+
+@include exports('skeleton-placeholder') {
+  .#{$prefix}--skeleton__placeholder {
+    @include skeleton;
```
Comment (asudoh) [inline src/components/skeleton/_skeleton-placeholder.scss:8]: Can we add `width`/`height` styles so the React variant can be used without in-line styles?

@alisonjoseph Does skeleton support `rem`-based dimensions?

Comment (ryanomackey) [reply src/components/skeleton/_skeleton-placeholder.scss:8]: @asudoh Could you expound on that? I agree that we should probably avoid in-line styles, but I'm curious what the alternative implementation might look like.

Comment (asudoh) [reply src/components/skeleton/_skeleton-placeholder.scss:8]: @ryanomackey Great that you asked - We can do something like:

```scss
.#{$prefix}--skeleton__placeholder {
  @include skeleton;
  width: rem(50px);
  height: rem(50px);
}
```

And consumers should be able to:

```scss
.bx--skeleton__placeholder.my--skeleton__placeholder {
  width: 100px;
  height: 100px;
}
```

And apply `my--skeleton__placeholder` class to the React component, etc. Let me know if you have any further questions. Thanks!

Comment (alisonjoseph) [reply src/components/skeleton/_skeleton-placeholder.scss:8]: Sorry for my late reply on this.  Yes, you can use rem for height/width. 

For the React component were you planning on setting the height/width there as props? Still wouldn't hurt to have a default in there as Akira is suggestings. Then a user could override with either css or the React component could override with inline css.

Comment (ryanomackey) [reply src/components/skeleton/_skeleton-placeholder.scss:8]: @asudoh @alisonjoseph Okay, so I added some default dimensions to the class, and then over in the [React PR](https://github.com/IBM/carbon-components-react/pull/1401), I got rid of the inline styles and added an example with custom classes to the storybook.

So, consumers can still use inline styles, but this should nudge them in a more performant direction 😉

**Review by asudoh — APPROVED** [review]
LGTM 👍 - Thanks @ryanomackey for the update!

**Review by alisonjoseph — APPROVED** [review]
Thanks! 👍

**Pull request merged**